<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScraperSettingsWidget</class>
 <widget class="QWidget" name="ScraperSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>625</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabScrapers">
      <attribute name="title">
       <string>Scrapers</string>
      </attribute>
      <layout class="QVBoxLayout" name="vLayout_MovieScrapers">
       <item>
        <widget class="ScraperSettingsTable" name="scraperSettings" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabCustomMovieScraper">
      <attribute name="title">
       <string>Custom Movie Scraper</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <widget class="QLabel" name="lblCustomMovieScraperHelp">
         <property name="text">
          <string>Combine multiple scrapers to your custom scraper. If you select other scrapers than IMDB, The Movie DB and Fanart.tv multiple searches may be necessary as only these three share an id.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="customScraperTable">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Item</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Scraper</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabCustomTvScraper">
      <attribute name="title">
       <string>Custom TV Scraper</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="CustomTvScraperSettingsWidget" name="customTvScraperSettings" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScraperSettingsTable</class>
   <extends>QWidget</extends>
   <header>ui/settings/ScraperSettingsTable.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CustomTvScraperSettingsWidget</class>
   <extends>QWidget</extends>
   <header>ui/settings/CustomTvScraperSettingsWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
